### PR TITLE
update prometheus client library

### PR DESCRIPTION
**Pull request by idoqo**
Updates prometheus client due to [CVE-2022-21698](https://www.whitesourcesoftware.com/vulnerability-database/CVE-2022-21698) which causes [failure in WhiteSource security checks](https://github.com/percona/pmm-agent/pull/325/checks?check_run_id=5396661370).

Build: SUBMODULES-2366

### CODE REVIEW

**Comment by artemgavrilov** [comment]
@idoqo CI is failning

File: actions/mysql_query_show_action_test.go [hunk 1]
```diff
@@ -47,13 +47,13 @@ func TestMySQLQueryShow(t *testing.T) {
...
 		assert.LessOrEqual(t, 15000, len(b))
-		assert.LessOrEqual(t, len(b), 26800)
+		assert.LessOrEqual(t, len(b), 27000)
 
 		data, err := agentpb.UnmarshalActionQueryResult(b)
 		require.NoError(t, err)
 		t.Log(spew.Sdump(data))
 		assert.LessOrEqual(t, 400, len(data))
-		assert.LessOrEqual(t, len(data), 700)
+		assert.LessOrEqual(t, len(data), 750)
```
Comment (artemgavrilov) [inline actions/mysql_query_show_action_test.go:56]: Hm, I wonder why this values changed?

**Review by artemgavrilov — COMMENTED** [review]
Let's create feature build to make sure that nothing was broken

File: actions/mysql_query_show_action_test.go (hunk 1 above)
Comment (artemgavrilov) [reply actions/mysql_query_show_action_test.go:56]: In PR that you mentioned all tests are green and there is no such changes it tests.

Comment (idoqo) [reply actions/mysql_query_show_action_test.go:56]: `make test-race` failed on the PR with this https://github.com/percona/pmm-agent/runs/5464274760?check_suite_focus=true

I should take some time to try and investigate further anyway.

**Review by artemgavrilov — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -28,16 +28,16 @@ require (
 	github.com/percona/percona-toolkit v3.2.1+incompatible
 	github.com/percona/pmm v0.0.0-20220310104949-dd33560bcdde
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.5.1
-	github.com/prometheus/common v0.10.0
+	github.com/prometheus/client_golang v1.11.1
```
Comment (ShashankSinha252) [inline go.mod:31]: Any reason to downgrade it from `v1.12.1` to `v1.11.1` ?

Comment (ShashankSinha252) [inline go.mod:31]: Any reason to downgrade it from `v1.12.1` to `v1.11.1` ?

Comment (idoqo) [reply go.mod]: The log package got removed from prometheus so we downgraded to keep it - but there's no reason why we can't remove it and directly use logrus too, so I've updated the code now.

File: actions/mysql_explain_action.go [hunk 3]
```diff
@@ -55,7 +55,7 @@ func NewMySQLExplainAction(id string, params *agentpb.StartActionRequest_MySQLEx
 	if params.TlsFiles != nil && params.TlsFiles.Files != nil {
 		err := tlshelpers.RegisterMySQLCerts(params.TlsFiles.Files)
 		if err != nil {
-			log.Error(err)
+			logrus.Error(err)
```
Comment (ShashankSinha252) [inline actions/mysql_explain_action.go:58]: Not sure if the move to logrus brings any UX change on binary output front. Let me know if this switch is okay.

**Review by ShashankSinha252 — COMMENTED** [review]
LGTM

File: actions/mysql_explain_action.go (hunk 3 above)
Comment (idoqo) [reply actions/mysql_explain_action.go:58]: Yeah I don't think it does either. This was the only place we used it and it wasn't customized in any way. Prometheus' common/log used to be a wrapper around logrus and has been deprecated for a while, it just got removed recently (https://github.com/prometheus/common/pull/306)

**Comment by BupycHuk** [comment]
@idoqo please fix merge conflicts

**Comment by idoqo** [comment]
> @idoqo please fix merge conflicts

Actually, `main` already has the updated package version so I'll go ahead and close this.

**Pull request closed without merging**